wrpclient/wrpclient_test.go

```diff
@@ -0,0 +1,152 @@
+/**
+ * Copyright 2022 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package wrpclient
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/xmidt-org/wrp-go/v3"
+)
+
+func TestCheckClientConfig(t *testing.T) {
+	defaultClient := Client{
+		URL:           "http://localhost:6200",
+		HTTPClient:    &http.Client{},
+		RequestFormat: 1,
```

Comment:
Whenever possible, I would suggest using enums rather than int values.  It makes the code more readable and easy to understand for someone less familiar with it.